compaction_test.go

```diff
@@ -118,10 +118,7 @@ func TestCompaction(t *testing.T) {
 	var err error
 	outputStream, err = outputstream.NewOutputStream("")
 
-	tempdir, err := ioutil.TempDir("", "robust-test-")
-	if err != nil {
-		t.Fatalf("ioutil.TempDir: %v", err)
-	}
+	tempdir := t.TempDir()
 	defer os.RemoveAll(tempdir)
```

Comment:
This line can be removed because t.TempDir() does cleanup.

Reply:
Made a new commit to remove that line.